core/src/test/java/io/questdb/cairo/AbstractCairoTest.java

```diff
@@ -69,6 +69,8 @@
     protected static int binaryEncodingMaxLength = -1;
     protected static CharSequence defaultMapType;
     protected static int pageFrameMaxSize = -1;
+    protected static int rndFunctionMemoryPageSize = -1;
```

Comment:
tiny case issue - should be rndFunctionMemory**M**axPages

Reply:
> tiny case issue - should be rndFunctionMemory**M**axPages

fixed